GoogleDataTransport/GDTCORLibrary/GDTCORReachability.m

```diff
@@ -98,22 +104,25 @@ - (instancetype)init {
       }
     });
   }
+#endif
   return self;
 }
 
-- (void)setCallbackFlags:(SCNetworkReachabilityFlags)flags {
+#if !TARGET_OS_WATCH
```

Comment:
For things like code completion, I'd prefer this to be inside the method body as opposed to the outside.

Reply:
Remove the if check